test/solver_tests/dissimilarity_functions_test.py

```diff
@@ -27,7 +27,7 @@ def setUp(self):
         }
 
         self.weights = solver_utilities.transform_priors(
-            self.priors, lambda x: -np.log(x)
+            self.priors, "negative_log"
         )
```

Comment:
can you add tests looking at the other prior transformations you added?

Also if there are errors that are expected to be thrown with certain inputs, can you please test that too?